### PR TITLE
Feature/add rpc anno

### PR DESCRIPTION
**Pull request by hellomyboy**
**What this PR does**:
https://github.com/mosn/layotto/issues/112
add comments to RPC related code
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by seeflood** [comment]
Thanks for your contribution!
@MoonShining  Could u help review this PR?

File: components/rpc/invoker/mosn/channel/connpool.go [hunk 1]
```diff
@@ -59,8 +62,8 @@ func (w *wrapConn) close() error {
 	return err
 }
 
-// im-memory fake conn pool
-func newConnPool(
+// NewConnPool im-memory fake conn pool
+func NewConnPool(
```
Comment (MoonShining) [inline components/rpc/invoker/mosn/channel/connpool.go:66]: why expose this function?

Comment (hellomyboy) [reply components/rpc/invoker/mosn/channel/connpool.go:67]: i add more complete explanation

Comment (MoonShining) [reply components/rpc/invoker/mosn/channel/connpool.go:67]: why change method name

Comment (hellomyboy) [reply components/rpc/invoker/mosn/channel/connpool.go]: It's amazing. In fact, I didn't change this. I may have pressed the shortcut key. I've changed it back

**Review by MoonShining — APPROVED** [review]

**Comment by seeflood** [comment]
Thanks again for your contribution,and welcome to the Layotto community!

**Pull request merged**